src/Controller/UI/SearchTermsReportController.php

```diff
@@ -226,10 +226,15 @@ protected function getDatav2(): array
                 $searchResults['1stScore'] = $result['payLoad']['elasticScoreFirstResult'] ?? '';
             }
 
+            if (empty($result['payLoad']['subSite'])) {
```

Comment:
Maybe `key_exists` is safer?

Reply:
Per https://www.php.net/manual/en/function.array-key-exists.php "array_key_exists() will search for the keys in the first dimension only. Nested keys in multidimensional arrays will not be found."